docs/docs/transactions.md

```diff
@@ -67,11 +74,11 @@ CLS works like a thread-local storage for callbacks. What this means in practice
 
 ```js
 sequelize.transaction(function (t1) {
-  namespace.get('transaction') === t1;
+  namespace.get('transaction') === t1; // prints true
```

Comment:
Doesnt actually print anything. Doesn't need the comment either, should be illustrative enough.